src/main/java/com/redislabs/Source.java

```diff
@@ -61,11 +61,16 @@ public Source(String connector, String name, String xmlDef) {
     this.name = name;
     this.xmlDef = xmlDef;
     this.propertyMappings = new HashMap<>();
-    
+
     StandardServiceRegistry tempRegistry = new StandardServiceRegistryBuilder()
-        .configure( InMemoryURLFactory.getInstance().build("configuration", RGHibernate.get(connector).getXmlConf()))
+         .configure( InMemoryURLFactory.getInstance().build("configuration", RGHibernate.get(connector).getXmlConf()))
         .build();
-    
+
+    //String dbpass = "1234";
+    //StandardServiceRegistry tempRegistry = new StandardServiceRegistryBuilder()
+    //        .configure( InMemoryURLFactory.getInstance().build("configuration", RGHibernate.get(connector).getXmlConf().replaceAll("(connection\\.password\">)[^<]*", "$1" + dbpass)))
+     //       .build();
+
```

Comment:
Remove?

Reply:
Place holder for getting a password from external source.

Reply:
Removed this for now until we figure out what is the based way to handle password from external source